### PR TITLE
Try a simple share -> mastodon via proxy

### PR DESCRIPTION
**Pull request by jaw6**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [x] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

Per Slack, it seems we're getting requests for a share-to-Mastodon feature. One small complicating factor with Mastodon is that there isn't any _one_ Mastodon, but there's a way around this via a [proxy](https://toot.kytta.dev/). If Mastodon becomes a _thing_, or if the proxy isn't reliable enough, we might want to revisit this to handle more of it on our end, but seems small enough to try it for now?

`cc` @erinb223 

## Added/updated tests?

- [ ] Yes
- [x] No, and this is why: all the interesting behavior is offsite
- [ ] I need help with writing tests

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

- [ ] I've updated the [Developer Docs](https://developers.forem.com) or
      [Storybook](https://storybook.forem.com/) (for Crayons components)
- [ ] This PR changes the Forem platform and our documentation needs to be
      updated. I have filled out the
      [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md)
      issue template so Community Success can help update the Admin Docs
      appropriately.
- [ ] I've updated the README or added inline documentation
- [ ] I've added an entry to
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
- [ ] I will share this change in a [Changelog](https://forem.dev/t/changelog)
      or in a [forem.dev](http://forem.dev) post
- [x] I will share this change internally with the appropriate teams
- [ ] I'm not sure how best to communicate this change and need help
- [ ] This change does not need to be communicated, and this is why not: _please
      replace this line with details on why this change doesn't need to be
      shared_

## [optional] What gif best describes this PR or how it makes you feel?

stampy

### CODE REVIEW

File: config/locales/views/actions/fr.yml [hunk 1]
```diff
@@ -15,6 +15,8 @@ fr:
           text: Partager sur Hacker News
         linkedin:
           text: Partager sur LinkedIn
+        mastodon:
+          text: Partager sur Mastodon
```
Comment (Ridhwana) [inline config/locales/views/actions/fr.yml:19]: Love your french! 😁

**Review by Ridhwana — APPROVED** [review]
Thanks Joshua! Looks good to me!

File: config/locales/views/actions/fr.yml (hunk 1 above)
Comment (jaw6) [reply config/locales/views/actions/fr.yml:19]: Un des nombreux avantages d'avoir vécu au Canada 🇨🇦

**Review by fdocr — APPROVED** [review]

**Comment by andypiper** [comment]
Adding a quick note here that there's a [share to Fediverse button](https://github.com/Uden-AI/fediverse-share) that wants the user to insert an instance URL in a drop down, which is another alternative to the proxy used in this solution.

**Comment by jaw6** [comment]
@andypiper Yep, that's a good resource if we decide we want this to handle this more on our side than with the proxy! :+1:

**Pull request merged**